src/models/vec_model.jl

```diff
@@ -57,33 +65,55 @@ function set_objective_multiple!(model::VecModel, m)
     return model
 end
 
-function optimize(model::VecModel, optimizer::AbstractOptimizer, x0, args...; kwargs...)
-    workspace = Workspace(model, optimizer, copy(x0), args...; kwargs...)
+"""
+Generic `optimize` for VecModel
+"""
+function optimize(model::VecModel, optimizer::AbstractOptimizer, x0::Nothing, args...; options=nothing, kwargs...)
```

Comment:
remove `options` from the kwargs and use `workspace = Workspace(model, optimizer, args...; kwargs...)` instead of this if statement

Reply:
> remove `options` from the kwargs and use `workspace = Workspace(model, optimizer, args...; kwargs...)` instead of this if statement

It's a messy part leads to many error, I redesigned it in my local version